components/nvs_flash/include/nvs.h

```diff
@@ -26,7 +26,7 @@ extern "C" {
 /**
  * Opaque pointer type representing non-volatile storage handle
  */
-typedef uint32_t nvs_handle;
+typedef uint32_t nvs_handle_t;
```

Comment:
We probably don't want to break all existing projects' code which uses nvs_handle, so at minimum we would want another typedef here to maintain compatibility.

Reply:
Yep that's a good point, added back the original typedefs I replaced